### PR TITLE
Further refined LwjglGraphics.checkOpenGL() method.

### PR DESCRIPTION
**Pull request by emanuele3d**
Mini PR as I realized that for some reason I didn't include OpenGL 2.0 in the checks even though it is used in the code.

I also added a comment to highlight that some of the extensions required eventually became part of Opengl 3.0.

### CODE REVIEW

File: engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java [hunk 1]
```diff
@@ -254,18 +254,18 @@ private void checkOpenGL() {
...
+                GLContext.getCapabilities().GL_ARB_half_float_pixel};   // http://en.wikipedia.org/wiki/OpenGL#OpenGL_3.0
 
         String[] capabilityNames = {"OpenGL12",
                                     "OpenGL14",
                                     "OpenGL15",
+                                    "OpenGL20",
                                     "GL_ARB_framebuffer_object",
                                     "GL_ARB_texture_float",
-                                    "GL_ARB_half_float_pixel",
-                                    "GL_ARB_shader_objects"};
```
Comment (emanuele3d) [inline engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java]: *GL_ARB_shader_objects* is part of OpenGL 2.0, hence the removal of the related lines.

**Comment by Cervator** [comment]
Thanks again for this logging, it is quickly becoming one of the most useful little snippets :-)

Example from just today: https://www.facebook.com/photo.php?fbid=733910276722085&set=o.248329655219905&type=1&permPage=1

Although it is hard to read the text in that screenshot I was able to get enough details for the video card to date it to 2006-2008 without any further questions. I'm sad for the user unable to play on such old hardware, but at least we can figure it out fast now.

I got the few Checkstyle fixes out of the way real quick since we were touching the file again

**Comment by emanuele3d** [comment]
Thank you for the screenshot. It's interesting because it turns out it's not quite mentioning the integrated graphics but only the parent chipset.

I'm thinking that irrespective of our capability to solve the problem we should probably add additional system information (i.e. Operating System) so that the user has all the information necessary to attempt a driver update. Not that it would have helped in this case. But I imagine that a kid might try and might get stuck and give up just because he's missing, say, if the OS is 32 or 64 bit. That information is already fetched elsewhere and it'd be duplicating some code, but it'd be beneficial for the user. 

Thanks for running Checkstyle, I completely forgot. =P

**Comment by Cervator** [comment]
No problem and agreed on more logging - add whatever makes sense :-)

We could probably also throw #1419 in there real quick sometime

**Pull request merged**